### ISSUE TITLE
Convert all current Unit Process designs to classes. 

### ISSUE THREAD

**Issue by ethan92429** [AguaClara/aguaclara#67]
Let's make sure that Sed, Filter and Entrance tank and any other template is brough up to speed with the flocculator class. @SageWS offered to work on some of this - maybe Sage could join the meetings with Fletcher and Zoe to get acquainted with all that involves. A good reference is how the LFOM class has changed. [Here is the old, Jinja method of loading components](https://github.com/AguaClara/aide_template/blob/YAML/templates/lfom.yaml) and [here is the corresponding new class](https://github.com/AguaClara/aide_render/blob/master/aide_render/templates/lfom.py) (with a lot more comments to explain what is happening. @fletchapin @zoemaisel which unit process do you think Sage should work on first? Maybe they can tackle one, and then y'all could meet to discuss it? what does everyone think of that?

**Comment by zoemaisel** [comment]
Yes, sounds like a good plan. @SageWS , Fletcher and I are planning to meet on Monday, May 14th from 3-5pm in the AguaClara design lab. Are you available to meet with us then?

**Comment by fletchapin** [comment]
@SageWS The entrance tank might be a good place to start because it's simple, but before we can create the class we have to modify the functions for the entrance tank.

I can let you know when those are modified and then you could take a look at the process or try to convert it if you'd like (but no need to if you'd like to wait)

**Comment by SageWS** [comment]
@fletchapin that works for me. Let me know when you're done with your piece and I'll see what I can get converted before Monday.

**Comment by fletchapin** [comment]
@SageWS Sorry it took me a while to get around to but I've updated the [entrance tank functions](https://github.com/AguaClara/aide_design/blob/filter_design/aide_design/functions/ent_tank.py)  within the filter_design branch.

What we did with the flocculator was essentially use the logic in the floc_agg function to build the different attributes of the flocculator, so we can do a similar thing with the ent_tank_agg function.

**Comment by eak24** [comment]
This still needs to be done and will be a major focus this semester.

**Comment by oliver-leung** [comment]
I'm offloading this issue to #39 and #48 , as well as any other issues for specific classes that we need to implement. Open a new issue to discuss any other plant components' design code!
